recognition/ISIC_skin_cancer/data_generator.py

```diff
@@ -0,0 +1,67 @@
+import os
```

Comment:
Code header block should be added for all scripts/source files

Reply:
Hi Shakes, 
I don't really get what do you mean about this comment. Could you please explain a bit more about it?